torchci/pages/api/flaky-tests/[testinfo].ts

```diff
@@ -0,0 +1,33 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import fetchFlakyTests from "lib/fetchFlakyTests";
+import { FlakyTestData } from "lib/types";
+
+interface Data {}
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  // Expect testinfo to be /testName%20testSuite%20testFile
+  const testinfo = req.query.testinfo;
+
+  let numHours = "336";  // representing 14 days = 14x24 hours
```

Comment:
nit: I think people do like `14 * 24 + ""` usually instead of putting a comment.